unit-tests/py/rspy/devices.py

```diff
@@ -73,8 +73,13 @@ def __init__( self, sn, dev ):
         self._physical_port = dev.supports( rs.camera_info.physical_port ) and dev.get_info( rs.camera_info.physical_port ) or None
 
         self._usb_location = None
+        self._is_dds = False
         try:
-            self._usb_location = _get_usb_location(self._physical_port)
+            if self._physical_port.startswith('realsense/'):
```

Comment:
I see we already have `dev` , let's also ask if the connection type is `dds`?

Reply:
This way we fixed the error on D457 too, currently I see
`-E- Failed to get usb location: invalid physical port '/dev/video-rs-depth-0'`
On D457 the connection type is GMSL